app/helpers/users_helper.rb

```diff
@@ -1,2 +1,8 @@
 module UsersHelper
+  # Returns the Gravatar for the given user.
+  def gravatar_for user, size: 80
```

Comment:
`80` giá trị này em thêm vào file settings nhé

nếu chưa có file settings thì em tìm hiểu `gem config in rails `nhé em